### ISSUE TITLE
Support for .rp9 files?

### ISSUE THREAD

**Issue by FrodeSolheim** [FrodeSolheim/fs-uae-launcher#5]
There hasn't been a major call for support of .rp9 files, but a couple of people have requested it.

Most likely, an implementation wouldn't be done in FS-UAE, but rather in FS-UAE Launcher - and an initial implementation could be relatively simple (read-only, possibly not support all features of the .rp9 files).

The launcher could index .rp9 files and add them to the configuration list, and also extract publisher / year / images from the .rp9 files for display in the launcher.

**Comment by MaddTheSane** [comment]
The Amiga Forever package/installer/disc includes them. Opening one via TextWrangler revealed it to be a compressed file (most likely zip) with disk images, an xml file, a png preview image, and a text file that describes it.

**Comment by FrodeSolheim** [comment]
@MaddTheSane Yes, I know, the RP9 format is documented here: http://www.retroplatform.com/reference/ :) (It is a zip file as you suspect it is)

**Comment by robsaunders** [comment]
I definitely support this idea - I am used to complicated software and found FS-UAE to be user-hostile when trying to set it up and support it. RP9s don't just contain the disks together they also have a configuration format all in one. If support for this is not planned, surely there must be SOME method of downloading a self contained game image? I mean it's kind of ridiculous to assume every single user will go through and load disks one by one every time...?

**Comment by wyan** [comment]
I was wondering if this might still happen. It'd be really cool to have an easy way to install / run these files :)

**Comment by dnouls** [comment]
It seems that some RP9 files are working. But many are not. I guess it depends on what is inside ? I managed to play some of the demos in the AmigaForever distribution, but many games fail to load. 
If the RP9 files contains ADF files it seems to work. If it contains HDF files it does not.
